### PR TITLE
apps/pano: Ugrade remix version to 1.7.5

### LINKED ISSUE #145: Upgrade pano's remix version
In order for us to complete #144 we need to upgrade remix versions. we could've done this later, but let's do it before it's late.

### PR DESCRIPTION
**Pull request by usirin**
Fixes #145

### CODE REVIEW

File: apps/pano/Dockerfile.dev [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
 
-FROM node:16 AS installer
+FROM node:16.14.2 AS installer
 WORKDIR /app
 COPY .gitignore .gitignore
 COPY --from=builder /app/out/json .
 COPY --from=builder /app/out/package-lock.json ./package-lock.json
-RUN npm install --workspace=@kampus-apps/pano
+RUN npm install
+RUN npm ls @remix-run/serve
```
Comment (cansirin) [inline apps/pano/Dockerfile.dev:14]: do we need this?

File: apps/pano/tsconfig.json [hunk 2]
```diff
@@ -4,7 +4,18 @@
     "baseUrl": ".",
     "paths": {
       "~/*": ["./src/*"]
-    }
+    },
+    "allowJs": true,
```
Comment (cansirin) [inline apps/pano/tsconfig.json:8]: Aren't these rules getting extended by "@kampus/tsconfig/remix.json"?

Comment (usirin) [reply apps/pano/tsconfig.json:8]: upgrading to remix auto generated all of these, these will be addressed in the kampus-ui pr.

Comment (cansirin) [reply apps/pano/tsconfig.json:8]: 💯

**Review by cansirin — APPROVED** [review]

**Pull request merged**